receiver/prometheusexecreceiver/factory.go

```diff
@@ -47,6 +48,7 @@ func createDefaultConfig() config.Receiver {
 	return &Config{
 		ReceiverSettings: config.NewReceiverSettings(config.NewComponentID(typeStr)),
 		ScrapeInterval:   defaultCollectionInterval,
+		ScrapeTimeout:    defaultTimeoutInterval,
```

Comment:
Is there a test that could be changed to ensure we don't have the same problem in the future?

Reply:
good question, I am not familiar with the code base but it could be enough to simply not explicitly define this parameter in the existing factory test.

so it will take the zero value and will trigger an error when running go test.

Reply:
I did not push the change yet ^^ will make another PR :rocket:

Reply:
here it is https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/7594